### PR TITLE
alter tables that create unique keys that are too long to have row_format DYNAMIC for pootle 2.7.2

### PR DESCRIPTION
**Pull request by fabada**
@ENuge @felfilali
Note this is being merged to stable/2.7.2. Go ahead and also merge to master if you'd like.

### CODE REVIEW

File: pootle/apps/pootle_app/migrations/0001_initial.py [hunk 1]
```diff
@@ -28,6 +28,12 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model, pootle.core.mixins.treeitem.CachedTreeItem),
         ),
+        migrations.RunSQL("ALTER TABLE `pootle_app_directory` ROW_FORMAT=DYNAMIC"),
```
Comment (ENuge) [inline pootle/apps/pootle_app/migrations/0001_initial.py]: Where did the name `pootle_app_directory` come from? Not sure if I'm missing something. I see `db_table` defines the name in the second case, not sure what does that here. (These same two questions apply in a few of the cases, but I figure it's the same answer each time.)

File: pootle/apps/pootle_app/migrations/0001_initial.py [hunk 2]
```diff
@@ -28,6 +28,12 @@ class Migration(migrations.Migration):
...
+        migrations.AlterField(
+            model_name='directory',
```
Comment (ENuge) [inline pootle/apps/pootle_app/migrations/0001_initial.py]: Is this case sensitive? (ie, should it be 'Directory'?)

File: pootle/apps/pootle_app/migrations/0001_initial.py (hunk 1 above)
Comment (fabada) [inline pootle/apps/pootle_app/migrations/0001_initial.py]: see https://docs.djangoproject.com/en/1.9/ref/models/options/#table-names
Basically django automatically derives a table name by {app_name}+{model_name}, unless specified by `db_table`

File: pootle/apps/pootle_app/migrations/0001_initial.py (hunk 2 above)
Comment (fabada) [inline pootle/apps/pootle_app/migrations/0001_initial.py]: I looked at (pootle/apps/pootle_store/migrations/0002_make_suggestion_user_not_null.py)[https://github.com/Yelp/pootle/blob/master/pootle/apps/pootle_store/migrations/0002_make_suggestion_user_not_null.py], which uses AlterField and it wasn't case sensitive, so I followed that. I'll confirm

**Comment by fabada** [comment]
ran migrations against a sandboxed db in my dev box and no errors popped up.

**Comment by ENuge** [comment]
Sweet!

**Pull request merged**